components/channel_header_dropdown/mobile_channel_header_dropdown.tsx

```diff
@@ -61,33 +64,22 @@ export default class MobileChannelHeaderDropdown extends React.PureComponent {
                         {dmHeaderIconStatus}
                         {this.getChannelTitle()}
                     </span>
-                    <FormattedMessage
-                        id='generic_icons.dropdown'
-                        defaultMessage='Dropdown Icon'
-                    >
-                        {(title) => (
-                            <span
-                                className='fa fa-angle-down header-dropdown__icon'
-                                title={title}
-                            />
-                        )}
-                    </FormattedMessage>
+                    <span
+                        className='fa fa-angle-down header-dropdown__icon'
+                        title={this.props.intl.formatMessage({id: 'generic_icons.dropdown', defaultMessage: 'Dropdown Icon'})}
+                    />
                 </a>
 
-                <FormattedMessage
-                    id='channel_header.menuAriaLabel'
-                    defaultMessage='Channel Menu'
-                >
-                    {(ariaLabel) => (
-                        <Menu ariaLabel={ariaLabel}>
-                            <ChannelHeaderDropdownItems isMobile={true}/>
-                            <div className='Menu__close visible-xs-block'>
-                                {'×'}
-                            </div>
-                        </Menu>
-                    )}
-                </FormattedMessage>
```

Comment:
Hey so why did we change, I have seen other places i coudnt find why anything else would break e2e, except for this 🤔